.github/workflows/main.yml

```
@@ -105,6 +106,8 @@ jobs:
             echo "## Install"
             echo "1. Download the rock."
             echo "2. Install it using luarock: \`luarock install <ROCK>.src.rock\`"
+            echo ""
+            echo "**Full Changelog**: https://github.com/DataDog/kong-plugin-ddtrace/compare/v{LATEST_RELEASE}...${GITHUB_SHA}"
```

```suggestion
            echo "**Full Changelog**: curl -s https://api.github.com/repos/DataDog/kong-plugin-ddtrace/compare/v{LATEST_RELEASE}...${GITHUB_SHA} \
  | jq -r '.commits[].commit.message'}"
```
Something like that would only show the commits message

---

There might be a misunderstanding. The goal is to generate a link with the changes from the `HEAD` to the latest release. Something like: https://github.com/DataDog/kong-plugin-ddtrace/releases/tag/v0.2.2

---

Alright, there was indeed a misunderstanding from my side